docs/tutorials/flight_controller_telemetry_tutorial.rst

```diff
@@ -0,0 +1,395 @@
+
+    MOT_POLE_COUNT to Properly Calculate RPM
+    
+After setting the MOT_POLE_COUNT to 200, the RPM values displayed in QGroundControl should be accurate.
+
+Standard PWM Telemetry
+***********************
+Vertiq modules support sending ESC telemetry when using :ref:`Standard PWM <hobby_standard_pwm>` or :ref:`other analog protocols <hobby_other_protocols>`
+as covered in the :ref:`ESC Telemetry <telemetry_analog_request>` section of the Feature Reference Manual. This telemetry is requested by sending a 
+:ref:`30 microsecond pulse <telemetry_analog_request>` to the module over the throttle line.  
+
+However, as of v1.13.3 PX4 flight controllers do not seem to support this method of requesting ESC telemetry according to the 
+`PX4 documentation <https://docs.px4.io/main/en/peripherals/esc_motors.html#pwm>`_. The relevant portion of the `PX4 documentation <https://docs.px4.io/main/en/peripherals/esc_motors.html#pwm>`_
+is reproduced below with the important sentence regarding ESC telemetry highlighted.
+
+.. figure:: ../_static/tutorial_images/fc_telemetry_tutorial/px4_no_pwm_telemetry.png
+    :align: center
+    :width: 60%
+    :alt: PX4 Documentation on ESC Telemetry When Using Standard PWM
+
+    PX4 Documentation on ESC Telemetry When Using Standard PWM
+
+DroneCAN Telemetry
+******************
+First, follow the steps in the :ref:`tutorial for integrating a Vertiq module using DroneCAN with a PX4 flight controller <dronecan_fc_tutorial>`.
+After that tutorial, you should be able to control your module from the flight controller using DroneCAN. After completing that set up, no additional
+configuration of the flight controller is required for it to receive ESC telemetry. As covered in the :ref:`ESC Telemetry <telemetry_dronecan>` 
+section of the Feature Reference Manual, Vertiq modules constantly broadcast ESC telemetry messages on the DroneCAN bus when they are connected, and PX4
+flight controllers are able to accept those messages.
+
+The only additional configuration that may be useful is to adjust the :ref:`DroneCAN telemetry frequency <telemetry_dronecan_telemetry_frequency>` on
+your Vertiq module to change how frequently telemetry updates are broadcast.
+
+You can confirm that you are receiving ESC telemetry by using the `instrument panel <https://docs.qgroundcontrol.com/master/en/FlyView/FlyView.html#instrument_panel>`_ 
+in QGroundControl to display live ESC Status values. For the example shown below, the instrument panel was configured to show Rpm1, Voltage1, and Current1
+for ESCStatus, in order to show the RPM, Voltage, and Current values from the telemetry received from the module the flight controller considers Motor 1.
+If your module is powered and connected to the flight controller properly, live values for these data points should be shown in the instrument panel as shown below.
+
+.. figure:: ../_static/tutorial_images/fc_telemetry_tutorial/px4_dronecan_instrument_panel.png

```

Comment:
were you spinning this by hand? Why is there an RPM with no throttle applied?

Reply:
A throttle was applied, by the actuator test. The instrument panel just always shows 0% throttle when you are using the motor testing tools, it only shows throttles when you are actually applying them like for flight